### PR TITLE
Join by url (take 2)

### PR DESCRIPTION
**Pull request by ijt**
This resolves the conflicts in join-by-url. git is not allowing a push to join-by-url now, so this is a workaround. No doubt there is a cleaner way to do this, but we want to get this out now.

### CODE REVIEW

File: src/js/app/controllers.js [hunk 1]
```diff
@@ -10,12 +10,12 @@ require('./controllers/post/PostCtrl')(angularModule)
 require('./controllers/post/PostListCtrl')(angularModule)
 require('./controllers/post/CommentsCtrl')(angularModule)
 
-require('./controllers/community/CommunityCtrl')(angularModule)
-require('./controllers/community/CommunitySettingsCtrl')(angularModule)
-require('./controllers/community/CommunityPostsCtrl')(angularModule)
-require('./controllers/community/CommunityMembersCtrl')(angularModule)
-require('./controllers/community/CommunityInviteCtrl')(angularModule)
-require('./controllers/community/NewCommunityCtrl')(angularModule)
+require('./controllers/community/CommunityCtrl')(angularModule);
```
Comment (levity) [inline src/js/app/controllers.js]: this file looks like it had some accidental overwrites during merging -- 1) semicolons here

File: src/js/app/controllers.js [hunk 2]
```diff
@@ -31,11 +31,11 @@ angularModule
 .controller('NavCtrl', require('./controllers/NavCtrl'))
 .controller('CommunityEventsCtrl', require('./controllers/community/CommunityEventsCtrl'))
 .controller('JoinCommunityCtrl', require('./controllers/community/JoinCommunityCtrl'))
-.controller('PostCardCtrl', require('./controllers/post/PostCardCtrl'))
-.controller('WelcomePostCtrl', require('./controllers/post/WelcomePostCtrl'))
-.controller('ProfileEditCtrl', require('./controllers/profile/ProfileEditCtrl'))
```
Comment (levity) [inline src/js/app/controllers.js]: 2) ProfileEditCtrl is missing

File: src/html/ui/community/settings.tpl.html [hunk 3]
```diff
@@ -6,7 +6,7 @@
   <a class="navbar-right close" ng-click="close()">&times;</a>
 </nav>
 
-<div class="main-content">
+  <div class="main-content">
```
Comment (levity) [inline src/html/ui/community/settings.tpl.html]: indentation

File: src/css/community.less [hunk 4]
```diff
@@ -135,3 +135,14 @@ body.community {
     }
   }
 }
+
+.login-signup-modal .modal-content {
```
Comment (levity) [inline src/css/community.less]: looks like this was copied from project.less -- can you just remove it from there and here and add it to the bottom of entrance.less?

File: src/js/app/controllers.js (hunk 1 above)
Comment (ijt) [inline src/js/app/controllers.js]: Okay, I removed a bunch of semicolons.

File: src/js/app/controllers.js (hunk 2 above)
Comment (ijt) [inline src/js/app/controllers.js]: Added it back in.

File: src/html/ui/community/settings.tpl.html (hunk 3 above)
Comment (ijt) [inline src/html/ui/community/settings.tpl.html]: Fixing.

File: src/css/community.less (hunk 4 above)
Comment (ijt) [inline src/css/community.less]: Done.

**Comment by levity** [comment]
:cool:, :ship: it

**Comment by edwardwest** [comment]
What about the settings line in the moderator settings?

It looks like it's not there?

Thanks for working on this!
Edward

_______________________
*Edward West*
mobile 415.225.7335
Facebook <http://www.facebook.com/edwardjwest>  Twitter
<http://www.twitter.com/edwardwest>  LinkedIn
<http://www.linkedin.com/in/edwardwest>

**Comment by ijt** [comment]
Hey Edward, Lawrence just deployed it. Can you check again?

**Comment by edwardwest** [comment]
live now-- thank you!

_______________________
*Edward West*
mobile 415.225.7335
Facebook <http://www.facebook.com/edwardjwest>  Twitter
<http://www.twitter.com/edwardwest>  LinkedIn
<http://www.linkedin.com/in/edwardwest>

**Comment by edwardwest** [comment]
Hey Issac + Lawrence- I just tested this-- it showed me when I tried it as
a non logged in user the "sign in" modal, rather than the "sign up" modal
-- 

both have the other option, but can we reverse this? I'd like this join by
URL to prioritize Sign Ups as the first action--

Let me know if you can fix this quickly if possible--

Thank you!
Edward

_______________________
*Edward West*
mobile 415.225.7335
Facebook <http://www.facebook.com/edwardjwest>  Twitter
<http://www.twitter.com/edwardwest>  LinkedIn
<http://www.linkedin.com/in/edwardwest>

**Comment by levity** [comment]
Deploying a fix now

**Comment by edwardwest** [comment]
the other confusing thing is that it talks about how you need an invite
code in the sign up link, when it would be better if it said something
about the community you were about to join, but this can be a rev 1.5 fix

_______________________
*Edward West*
mobile 415.225.7335
Facebook <http://www.facebook.com/edwardjwest>  Twitter
<http://www.twitter.com/edwardwest>  LinkedIn
<http://www.linkedin.com/in/edwardwest>

**Comment by edwardwest** [comment]
thanks for jamming this out

_______________________
*Edward West*
mobile 415.225.7335
Facebook <http://www.facebook.com/edwardjwest>  Twitter
<http://www.twitter.com/edwardwest>  LinkedIn
<http://www.linkedin.com/in/edwardwest>

**Pull request merged**